src/vl-mapactions-delete-action.js

```diff
@@ -52,6 +53,7 @@ export class VlDeleteAction extends VlBoxSelectAction {
       }
     }, {
       style: style,
+      filter: filter
```

Comment:
Gewoon `filter` ipv `filter: filter`.
De const die op lijn 27 wordt aangemaakt lijkt me ook overbodig. Als je in het object `filter: options.filter` zet zal hij gewoon de filter of undefined teruggeven. Als hij erover valt dat options niet bestaan kan je zeggen `filter: options && options.filter`.

Idem style!